### PR TITLE
Use latest-or-auto

### PR DESCRIPTION
**Pull request by mmitche**
Latest now contains mono

### CODE REVIEW

**Comment by mmitche** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -9,7 +9,7 @@ def branch = GithubBranchName
 // What this repo is using for its machine images at the current time
 def imageVersionMap = ['Windows_NT':'latest-or-auto-dev15-rc',
                        'OSX':'latest-or-auto',
-                       'Ubuntu14.04':'20170728',
+                       'Ubuntu14.04':'latest-or-auto',
                        'Ubuntu16.04':'20170731']
```
Comment (rainersigwald) [inline netci.groovy]: Same for 16.04, or not yet?

**Comment by mmitche** [comment]
@rainersigwald Not yet.  You can actually leave this PR open for a bit.

**Comment by mmitche** [comment]
Actually, merge this since it's currently blocking runs.

**Comment by rainersigwald** [comment]
Ok :)

**Pull request merged**